### ISSUE TITLE
Unable to pass flake inputs to `darwin-configuration.nix`

### ISSUE THREAD

**Issue by tmillr** [LnL7/nix-darwin#694]
The docs (the README) say:

> Inputs from the flake can also be passed to darwinSystem, these inputs are then accessible as an argument, similar to pkgs and lib inside the configuration.
> 
> ```nix
> darwin.lib.darwinSystem {
>   system = "x86_64-darwin";
>   modules = [ ... ];
>   inputs = { inherit darwin dotfiles nixpkgs; };
> }
> ```

however this doesn't seem to be working for me.

Here's my flake:

```nix
  outputs = {
    self,
    nix-darwin,
    nixpkgs,
    neovim,
  }: {
    # darwinConfigurations."MacBook-Pro-3" = nix-darwin.lib.darwinSystem ((import nixpkgs {}).lib.debug.traceValSeqN 3 rec {
    darwinConfigurations."MacBook-Pro-3" = nix-darwin.lib.darwinSystem {
      system = "aarch64-darwin";
      modules = [./darwin-configuration.nix];
      inputs = {
        # neovim = neovim.packages.aarch64-darwin.neovim;
        inherit neovim;
        inherit nixpkgs;
        darwin = nix-darwin;
      };
    };
  };
}
```

Now when I build using this flake, the `neovim` attribute does not get passed as an argument to the `./darwin-configuration.nix` module, and therefore it errors with `error: attribute 'neovim' missing`.

> **Note** `neovim` is just a custom flake of mine and is part of my flake registry. This attribute is not null in the flake output above.

A trace of the argument to `./darwin-configuration.nix` shows that, indeed, a `neovim` attribute is not present/getting passed:

```sh
trace: { config = <CODE>; lib = <CODE>; modulesPath = <CODE>; options = «potential infinite recursion»; pkgs = <CODE>; specialArgs = { modulesPath = <CODE>; }; }`
```

**Comment by tmillr** [comment]
After several hours I think I found the problem. Essentially, I was referring to `neovim` in my `./darwin-configuration.nix`...but apparently it needs to be `inputs.neovim` instead. I believe the relevant lines that explain the reason for this are here: https://github.com/LnL7/nix-darwin/blob/7c16d31383a90e0e72ace0c35d2d66a18f90fb4f/eval-config.nix#L20-L22

Perhaps the documentation could be improved to note that the passed inputs are only accessible under the attribute named `inputs`.

**Comment by jonpulsifer** [comment]
@tmillr looks like the documentation around this was updated recently in https://github.com/LnL7/nix-darwin/pull/689, is that sufficient?

if so you can close this issue 😄

**Comment by tmillr** [comment]
Fixed by #689
